index.d.ts

```diff
@@ -2524,14 +2525,46 @@ declare namespace Eris {
   }
 
   export class VoiceConnection extends EventEmitter implements SimpleJSON {
-    channelID: string;
+    bitrate: number;
+    channelID: string | null;
+    channels: number;
     connecting: boolean;
-    current?: VoiceStreamCurrent;
+    connectionTimeout: NodeJS.Timeout | null;
+    current?: VoiceStreamCurrent | null;
+    ended?: boolean;
+    endpoint: URL;
+    frameDuration: number;
+    frameSize: number;
+    heartbeatInterval: NodeJS.Timeout | null;
     id: string;
+    mode?: string;
+    modes?: string;
+    opus: { [s: string]: unknown }; // TODO
+    opusOnly: boolean;
     paused: boolean;
+    pcmSize: number;
+    piper: Piper;
     playing: boolean;
     ready: boolean;
+    receiveStreamOpus?: VoiceDataStream | null;
+    receiveStreamPCM?: VoiceDataStream | null;
+    reconnecting: boolean;
+    samplingRate: number;
+    sendBuffer: Buffer;
+    sendNonce: Buffer;
+    sequence: number;
+    seret: Buffer;
+    shard: Shard | Record<string, never>;
+    shared: boolean;
+    speaking: boolean;
+    ssrc?: number;
+    ssrcUserMap: { [s: number]: string };
+    timestamp: number;
+    udpIP?: string;
+    udpPort?: number;
+    udpSocket: unknown | null; // TODO Dgram.Socket
```

Comment:
Install typings?

Reply:
`dgram` is stdlib, `@types/node` is already a dep

Reply:
Ah didn't know that